src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementBackedDocumentsService.java

```diff
@@ -21,28 +23,38 @@
 
     private final ClaimService claimService;
     private final DocumentManagementService documentManagementService;
+    private final ClaimIssueReceiptService claimIssueReceiptService;
     private final LegalSealedClaimPdfService legalSealedClaimPdfService;
     private final DefendantResponseCopyService defendantResponseCopyService;
+    private final DefendantResponseReceiptService defendantResponseReceiptService;
     private final CountyCourtJudgmentPdfService countyCourtJudgmentPdfService;
     private final SettlementAgreementCopyService settlementAgreementCopyService;
 
     @Autowired
     public DocumentManagementBackedDocumentsService(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS00107)